app/assets/javascripts/application.js

```diff
@@ -11,4 +11,5 @@
 // about supported directives.
 //
 //= require rails-ujs
-//= require bootstrap
```

Comment:
Add this back so that Bootstrap JS works. Also include jquery-rails

Reply:
Done